### PR TITLE
Add logic for saving every order to json-server.

### PR DESCRIPTION
**Pull request by olhavalchak**

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -171,5 +171,61 @@
         }
       ]
     }
+  ],
+  "orders": [
+    {
+      "userMail": "olia.valchak@gmail.com",
```
Comment (AndreiYar) [inline db.json:177]: userEmail

File: db.json [hunk 2]
```diff
@@ -171,5 +171,61 @@
...
+      "shopItems": [
+        {
+          "orderId": 1,
```
Comment (AndreiYar) [inline db.json:180]: shopItemId

File: src/api/api.js [hunk 3]
```diff
@@ -4,10 +4,25 @@ import { root } from './config';
...
   getQuestions: async (search) => {
     const res = await axios.get(`${root}/questions${search}`);
+
     return res.data;
   },
+  postOrders: async (userEmail) => {
+    const pushShopItems = () => {
+      const cartList = JSON.parse(localStorage.getItem('cartList'));
+      const userList = [];
+      cartList.forEach((item) => userList.push({ orderId: item.id, amount: item.amount }));
```
Comment (AndreiYar) [inline src/api/api.js:19]: use map

File: src/api/api.js [hunk 4]
```diff
@@ -4,10 +4,25 @@ import { root } from './config';
...
     const res = await axios.get(`${root}/shopItems${search}`);
+
     return res.data;
   },
   getQuestions: async (search) => {
     const res = await axios.get(`${root}/questions${search}`);
+
     return res.data;
   },
+  postOrders: async (userEmail) => {
```
[the remark this answers is not in the document]
Comment (olhavalchak) [reply src/api/api.js:15]: done

File: db.json (hunk 1 above)
Comment (olhavalchak) [reply db.json:177]: done

File: db.json (hunk 2 above)
Comment (olhavalchak) [reply db.json:180]: done

File: src/api/api.js (hunk 3 above)
Comment (olhavalchak) [reply src/api/api.js:19]: done

File: src/api/api.js [hunk 5]
```diff
@@ -4,10 +4,25 @@ import { root } from './config';
...
+    const pushShopItems = () => {
+      const cartList = JSON.parse(localStorage.getItem('cartList'));
+      const userList = [];
+      cartList.forEach((item) => userList.push({ orderId: item.id, amount: item.amount }));
+      return userList;
+    };
+
+    const order = { userMail: userEmail, shopItems: pushShopItems() };
```
[the remark this answers is not in the document]
Comment (olhavalchak) [reply src/api/api.js:23]: done

**Pull request merged**